lib/statsd/instrument.rb

```diff
@@ -336,31 +335,38 @@ def backend
     @backend ||= StatsD::Instrument::Environment.default_backend
   end
 
-  # Emits a duration metric.
+  # @!method measure(name, value = nil, sample_rate: nil, tags: nil, &block)
+  # @!method measure(name, value, sample_rate: nil, tags: nil)
```

Comment:
Is this macro actually meant to specify overloads?  Given that you are using `@overloads` below.  Or will this second method macro replace the first?

Reply:
After doing some experimentation with overloaded methods:

- The `@!method` macro ends up in the method summary list, so we should only specify one: 
<img width="709" alt="Screen Shot 2019-09-27 at 2 56 14 PM" src="https://user-images.githubusercontent.com/15870/65794952-2f07c500-e137-11e9-85d2-d85a0bf1660f.png">

- The `@overload` signatures do not show up in the method summary list, but will show up under the method details:

<img width="651" alt="Screen Shot 2019-09-27 at 2 55 41 PM" src="https://user-images.githubusercontent.com/15870/65795029-578fbf00-e137-11e9-85ee-36a2fdc09fc2.png">

I will update it so we only use `@!method` once for the overloaded metric.